Jenkinsfile

```diff
@@ -1,5 +1,7 @@
 library 'pipeline-utils@master'
 
+CCV = ""
```

Comment:
Could you please document this line so it's clearer what `CCV` is used for?

Reply:
Makes sense , i will try to get with Chuck or Tyler , to make more sense of it (https://drive.google.com/file/d/1HVt74gK4d-JJqo8d7sKmnrRba7viftvY/view)(@43:00) they had given the Architecture group a demo .